Jenkinsfile

```diff
@@ -23,6 +26,28 @@ pipeline{
 				}
 			}
 		}
+		// Download the two orthology files generated by PANTHER used to create the Reactome orthology mappings.
+		stage('Setup: Download Ortholog files from PANTHER'){
+			steps{
+				script{
+					sh "wget -q ftp://ftp.pantherdb.org/ortholog/current_release/Orthologs_HCOP.tar.gz"
+					sh "tar -xvf Orthologs_HCOP.tar.gz"
+					sh "wget -q ftp://ftp.pantherdb.org/ortholog/current_release/QfO_Genome_Orthologs.tar.gz"
+					sh "tar -xvf QfO_Genome_Orthologs.tar.gz"
```

Comment:
It might be more trouble than it's worth, but since the URL paths and file names are reused could they be put into variables so if they change the update(s) happen only in one place?

Reply:
No problem thats simple to do